fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -614,4 +617,37 @@ private static URI checksumURI( final String checksum ) {
         }
         return null;
     }
+
+    protected void createFixityEvent(final EventBus eventBus, final RdfStream rs,
+                                     final String baseURL, final String agent,
+                                     final String userID) {
+        final FixityEvent fixityEvent = new FixityEvent();
+        fixityEvent.setDate(System.currentTimeMillis());
+        fixityEvent.setType(128);
+        final StmtIterator it = rs.asModel().listStatements();
+        final HashMap<String,String> info = new HashMap<String,String>();
+        while (it.hasNext()) {
+           final Statement st = it.next();
+           if (st.getPredicate().toString().equals("http://www.loc.gov/premis/rdf/v1#hasFixity")) {
```

Comment:
For clarity and brevity please pull things like `st.getPredicate().toString()` into local variables instead of repeating them 4 times.